Makefile.toml

```diff
@@ -20,12 +20,28 @@ TARGET = "--target=x86_64-unknown-linux-musl"
 RUSTFLAGS="-C codegen-units=1 -C panic=abort -C lto=fat -C opt-level=3"
 VERBOSE = "-vv"
 
+[env.'release+static+aarch64']
+CC_aarch64_unknown_linux_musl="clang"
+AR_aarch64_unknown_linux_musl="llvm-ar"
+CFLAGS_aarch64_unknown_linux_musl="-I/usr/aarch64-linux-gnu/include"
```

Comment:
Is this ok?

I'd think for a musl build we'd want to point to something like `/usr/aarch64-linux-musl/include`.

Reply:
I tested this one out locally and it does work in a linux docker container which might mean that this flag is being ignored.

Reply:
If you run file or ldd commands against the binary that should tell you if it was fully statically linked (uses musl libc)

Reply:
I think these are libc independent headers?  It wasn't needed for me on my Linux machine, but didn't work in CI without it.  This should be part of the "sysroot" which should be discovered automatically, but isn't for some reason on GitHub actions.

Reply:
Does a musl variant of that path even exist?

Reply:
> I tested this one out locally and it does work in a linux docker container which might mean that this flag is being ignored.

It's definitely not ignored, with verbose output you can see the flag being included in compile commands.

Reply:
> I think these are libc independent headers?  It wasn't needed for me on my Linux machine, but didn't work in CI without it.  This should be part of the "sysroot" which should be discovered automatically, but isn't for some reason on GitHub actions.

Maybe not libc independent, but there's no equivalent package for musl cross compilation: https://packages.ubuntu.com/bionic/libc6-dev-arm64-cross so I don't know why it works on my machine.

Reply:
Most of this is from https://github.com/briansmith/ring/issues/1414#issuecomment-1055177218 but looks like ring itself doesn't have issues with the sysroot, so not sure why we do

Reply:
🤔 . I do have a musl variant on my path at `/usr/include/x86_64-linux-musl/`. (not cross compiling obv).

I guess I was just concerned that if those flags were necessary you might've not actually been linking musl libc, but joe tested the binary and it's fully statically linked.

Reply:
> 🤔 . I do have a musl variant on my path at `/usr/include/x86_64-linux-musl/`. (not cross compiling obv).
> 

Right, but there doesn't appear to be a aarch64 variant of this path

> 
> 
> I guess I was just concerned that if those flags were necessary you might've not actually been linking musl libc, but joe tested the binary and it's fully statically linked.

Could strace the whole compile process locally to figure out where the "bits" files are coming from